relay-general/src/store/normalize.rs

```diff
@@ -221,12 +236,42 @@ fn normalize_breakdowns(event: &mut Event, breakdowns_config: Option<&Breakdowns
     }
 }
 
-/// Ensure measurements interface is only present for transaction events
-fn normalize_measurements(event: &mut Event) {
+/// Enforce the limit on custom (user defined) measurements.
```

Comment:
nit / personal opinion: the function accepts a `Measurements` object that may contain built-in measurements (in fact, the function does get built-in measurements, and that's the first thing checked to decide if we should keep a measurement or not), so it's not really enforcing the limit on custom measurements but on all measurements. Personally, I'd generalize this line, and mention in another paragraph that all built-in measurements are accepted and we limit the custom ones by the project config.
```suggestion
/// Enforce the limit on measurements.
```

Reply:
I renamed the function and updated the doc comment to clarify what it does.